docs/changelog.rst

```diff
@@ -6,5 +6,5 @@ Full Changelog
 
 .. changelog::
    :towncrier: ../
-   :towncrier-skip-if-empty:
+   :towncrier-skip-if-empty: True
```

Comment:
please revert this change. We *do* need a changelog fragment for a bug fix, see `docs/changes/README.rst`